metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala

```diff
@@ -379,7 +383,7 @@ case class ScalafixProvider(
       }
 
     val result = for {
-      api <- getScalafix(scalaBinaryVersion)
+      api <- getScalafix(scalaTarget.scalaVersion)
```

Comment:
https://github.com/scalacenter/scalafix/commit/14982001eeb52c8c46a3cbfabbddc4165abd1a1d#diff-8f1370021f0c21599d22a292e6ee5a9446f05c18a05f5c3bb250dea3884c72ee

Reply:
Keeping 2.13 by default for community rules via https://github.com/scalameta/metals/blob/f1ba8c5fa17a09c3fd68568f1d0dea4374a9e45f/metals/src/main/scala/scala/meta/internal/metals/ScalafixProvider.scala#L343-L344 - they can be loaded by cli_3